docs/20_smart-contracts/10_getting-started/10_dune-guide.md

```diff
@@ -2,153 +2,150 @@
 title: DUNE
 ---
 
-[Docker Utilities for Node Execution (DUNE)](https://github.com/AntelopeIO/DUNE) is a client tool that allows blockchain developers and node operators to perform boilerplate tasks related to smart contract development and node management functions.
+[Docker Utilities for Node Execution (DUNE)](https://github.com/AntelopeIO/DUNE) is a client tool. 
+As a blockchain developer or node operator, use DUNE for your common tasks.
 
-Before getting started with smart contract development, you need to learn about DUNE and how to install it on your platform.
+Before you begin, learn about DUNE and how to install it on your platform.
 
 ### Installation
 
-DUNE can be installed and run on the following platforms:
+You can install and run DUNE n the following platforms:
 * Linux
 * Windows
 * MacOS
 
-Installation instructions for each supported platform are available on the [DUNE's github project](https://github.com/AntelopeIO/DUNE) page.
+Each platform has their own installation instructions. Click [DUNE's GitHub project](https://github.com/AntelopeIO/DUNE) to view the instructions.
 
-Once you're done, you can run `dune --help` to see a list of all supported commands.
+Run `dune --help` to view a list of all supported commands.
 
 ## Wallets
 
-DUNE handles wallet management for you so that you don't have to. 
+DUNE handles wallet management for you so you don't have to. 
 
 If you need to import a new key into your wallet:
 
 ```shell
 dune --import-dev-key <PRIVATE_KEY>
 ```
 
-## Node Management
+## Node management
 
-Creating a new local EOS blockchain is easy with DUNE.
+DUNE easily creates a new local EOS blockchain.
+
+The following command creates a new node called `NODE_NAME`.  This node initiates with default settings. These default settings configure the node to serve as an API/producer node. You can deploy smart contracts to it and perform tests on it.
```

Comment:
This flowed better with the explanation text below the code block. 

The commands/code should always be the primary, and the explanation of the code is the backup just in case the reader didn't understand what's happening from the code line itself. We get to the code as quickly as possible.